out/WIPL.asm

```diff
@@ -714,7 +714,10 @@ WaitForHawkCmdCompletion:
 0502:    49                     sub BL, AL
 0503:    15 f6                  bnz WaitForHawkCmdCompletion
 0505:    09                     ret
-0506:    00
+
+AbortHandler:
+    ; We set IPL14 vector to this point
+0506:    00                     HALT
```

Comment:
They probably put the abort HALT here so the pattern on the address LEDs is distinctive.

Reply:
Yes, and note the location will be different depending on what stage it happens in, because later in the process this whole code gets relocated to upper RAM, and vector is also moved accordingly

Reply:
Which is clever. 
Though there are only two stages, before and after relocation. I guess it's possible for the top-of-ram scan to trigger a halt?